lib/sportradar/api/nfl/stat.rb

```diff
@@ -0,0 +1,7 @@
+module Sportradar
```

Comment:
Are we using this anywhere anymore, or did we shy away in favor of statpack & subclasses?